src/controllers/ExpenseController.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const ExpenseModel = require('../models/Expense');
+const UserModel = require('../models/User');
+
+const get = (req, res) => {
+  const params = req.query;
+
+  const expenses = ExpenseModel.get(params);
+
+  res.send(expenses);
+};
+
+const getById = (req, res) => {
+  const { id } = req.params;
+
+  if (id) {
+    const expense = ExpenseModel.getById({ id: +id });
+
+    if (expense) {
+      res.send(expense);
+
+      return;
+    }
+
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.sendStatus(400);
+};
+
+const create = (req, res) => {
+  const { title, amount, category, note, spentAt, userId } = req.body;
+
+  const user = UserModel.getById({ id: userId });
+
+  if (
+    ![title, amount, category, spentAt]
+      .some(item => item === undefined)
+    && user
+  ) {
+    const newExpense = ExpenseModel.create({
+      title,
+      amount,
+      category,
+      spentAt,
+      userId,
+      note: note || '',
+    });
+
+    res.status(201).send(newExpense);
+
+    return;
+  }
+
+  res.sendStatus(400);
+};
+
+const remove = (req, res) => {
+  const { id } = req.params;
+  const expenseIndex = ExpenseModel.remove({ id: +id });
```

Comment:
Would be better to check if the resource exists or not. If not return the error with status code 404, otherwise delete the resource and return the status with some message (optional)